client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -107,14 +111,18 @@ def prediction_url(self, value):
             raise ValueError("not a valid prediction_url")
         self._prediction_url = parsed.geturl()
 
+    def batch_prediction_url(self):
+        # Adding the v1/ to make sure we don't accidentally replace the wrong part of the URL
+        return self.prediction_url.replace("v1/predict", "v1/batch-predict")
```

Comment:
This seems hacky 🤔 

```suggestion
        return self.prediction_url.replace("/predict/", "/batch-predict/")
```

might be a bit less hacky (so we don't rely on the `/v1` routing)